### PR TITLE
 Refactor registrations #5670 - Partial form

### PR DESCRIPTION
**Pull request by giniouxe**

### CODE REVIEW

File: app/views/registrations/_form.haml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+				- if AppConfig.settings.captcha.enable?
+					= show_simple_captcha :object => 'user', :code_type => 'numeric'
+
+				= invite_hidden_tag(invite)
+
+			- if AppConfig.settings.terms.enable?
+				%p#terms.text-center
+					= t('registrations.new.terms', terms_link: link_to(t('registrations.new.terms_link'), terms_path, target: "_blank")).html_safe
+				= f.submit t('.sign_up'), class: "btn btn-block btn-large", data: {disable_with: t('.submitting')}
```
Comment (svbergerem) [inline app/views/registrations/_form.haml]: The translation keys should be `registrations.new.sign_up` and `registrations.new.submitting`

**Comment by svbergerem** [comment]
Please use 2 spaces instead of a tab for indentation.

**Comment by giniouxe** [comment]
Hi,

Sorry for the tabs.
Just updated the commit.

File: app/views/registrations/_form.haml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+.span5
+  .text-center
+    %h2#pod-name
+      = AppConfig.settings.pod_name
```
Comment (jhass) [inline app/views/registrations/_form.haml]: I wonder if we even want this part in the partial or just the plain form.

Comment (svbergerem) [inline app/views/registrations/_form.haml]: I agree.

**Comment by giniouxe** [comment]
Pod name is back to the new template.

**Comment by svbergerem** [comment]
Looks good! :) Could you please squash the commits?

**Comment by giniouxe** [comment]
No pb. Just squashed them.

**Comment by svbergerem** [comment]
Thank you. :)

**Pull request merged**